C/ctx8Pruned.c

```diff
@@ -260,6 +260,11 @@ const unsigned char ctx8Pruned[] = {
 };
 
 const size_t sizeof_ctx8Pruned = sizeof(ctx8Pruned);
+const unsigned char ctx8Pruned_witness[] = {
+
+};
+
+const size_t sizeof_ctx8Pruned_witness = sizeof(ctx8Pruned_witness);
```

Comment:
In 1b85dc31d80d36dc012755e4369aeeac815476a6:

Over in rust-simplicity, our MSVC Windows CI seems to shit the bed on this. It issues a warning that `sizeof` returns 0 and then has an internal compiler error and crashes.

Is this legal C?

Reply:
I think it is not legal.

Reply:
Fixed in #246.